.travis.yml

```diff
@@ -117,4 +121,4 @@ deploy:
   - provider: script
     on:  # yamllint disable-line rule:truthy
       all_branches: true
-    script: ./deploy.sh
+    script: travis_wait ./deploy.sh
```

Comment:
This is probably not useful in the `deploy` section. There are no other jobs here, it is just a single one that runs after all jobs passed.

Reply:
the Travis will timeout. if it won't receive any output. it's better to put it here as a safer side.

Reply:
Dropped this change

Reply:
Ah, ok, I have seen those timeouts before. If this helps preventing them, that is good.